### PR TITLE
[enterprise-4.12] Update 4.12 docs per style guide updates

### PR DESCRIPTION
**Pull request by kelbrown20**
**Version:** 4.12

**Description:** 

**Previews:** 

4.10+
- [Selecting an installation method and preparing a cluster -> Support installation methods for different platforms ](https://57071--docspreview.netlify.app/openshift-enterprise/latest/installing/installing-preparing.html#supported-installation-methods-for-different-platforms)
- Installing a user provisioned cluster on bare metal 
  1. [Installing RHCOS by using PXE or IPXE booting ](https://57071--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-bare-metal.html#installation-user-infra-machines-pxe_installing-bare-metal)
  2. [Advanced RHCOS installation reference](https://57071--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-bare-metal.html#installation-user-infra-machines-advanced_installing-bare-metal) 
- Installing on-premise with Assisted installer 
  1. [Installing an on premise cluster using the Assisted installer ](https://57071--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_on_prem_assisted/installing-on-prem-assisted.html) 
- Installing a cluster on AWS in a restricted network with user provisioned infrastructure 
  1. [Accessing the RHCOS AMI with stream meta data](https://57071--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-restricted-networks-aws.html#installation-aws-ami-stream-metadata_installing-restricted-networks-aws)
  2. [RHCOS AMIs for the AWS infrastructure](https://57071--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-restricted-networks-aws.html#installation-aws-user-infra-rhcos-ami_installing-restricted-networks-aws) 
- Installing a cluster on AWS with customizations
  1. [Tested instance types AWS ARM](https://57071--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations.html#installation-aws-tested-machine-types_installing-aws-customizations)
  2. [Tested instance types AWS](https://57071--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations.html#installation-aws-arm-tested-machine-types_installing-aws-customizations) 

4.11+ 
- [Configuring multi-architecture compute machines on an OpenShift Cluster](https://57071--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/multi-architecture-configuration.html)
  1. [Creating an ARM64 boot image using the Azure image gallery](https://57071--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/multi-architecture-configuration.html#multi-architecture-creating-arm64-bootimage_multi-architecture-configuration) 
  2. [Adding a multi-architecture compute machine set to your cluster using the ARM64 boot image](https://57071--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/multi-architecture-configuration.html#multi-architecture-modify-machine-set_multi-architecture-configuration)
- [Troubleshooting operating system issues -> Investigating kernel crashes](https://57071--docspreview.netlify.app/openshift-enterprise/latest/support/troubleshooting/troubleshooting-operating-system-issues.html#investigating-kernel-crashes)
- [Deploying installer provisioned clusters on bare metal -> Prerequisites](https://57071--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-prerequisites.html)
- Installing a cluser on AWS using CloudFormation templates
  1. [Uploading a custom RHCOS AMI in AWS](https://57071--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-user-infra.html#installation-aws-upload-custom-rhcos-ami_installing-aws-user-infra)
  2. [Creating the control plane machines in AWS](https://57071--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-user-infra.html#installation-creating-aws-control-plane_installing-aws-user-infra)
  3. [Creating the worker nodes in AWS](https://57071--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-user-infra.html#installation-creating-aws-worker_installing-aws-user-infra)
- [CLI Tools -> Installing tkn](https://57071--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/tkn_cli/installing-tkn.html)

4.12+ 
- Installing a cluster on Azure with customizations
  1. [Tested instance types for Azure ](https://57071--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-customizations.html#installation-azure-tested-machine-types_installing-azure-customizations)
  2. [Tested instance types for Azure ARM ](https://57071--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-customizations.html#installation-azure-arm-tested-machine-types_installing-azure-customizations)
- [Installing a cluster on Azure with customization -> Selecting an Azure Marketplace image ](https://57071--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-customizations.html#installation-azure-marketplace-subscribe_installing-azure-customizations)
- [Help-based operators -> Updating Help-baled operator projects for Operator SDK 1.25.4](https://57071--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/helm/osdk-helm-updating-projects.html#osdk-upgrading-projects_osdk-helm-updating-projects)

### CODE REVIEW

File: modules/installation-user-infra-machines-static-network.adoc [hunk 1]
```diff
@@ -367,7 +367,7 @@ a|`--insecure-ignition`
 a|Allow Ignition URL without HTTPS or hash.
 
 a|`--architecture <name>`
-a|Target CPU architecture. Valid values are `x86_64` and `arm64`.
+a|Target CPU architecture. Valid values are `x86_64` and `aarch64`.
```
Comment (kelbrown20) [inline modules/installation-user-infra-machines-static-network.adoc:370]: Have to check this one

**Review by aleskandro — COMMENTED** [review]
/label qe-approved

**Comment by ShaunaDiaz** [comment]
@maxwelldb Can you please complete the peer review?

File: installing/installing-preparing.adoc [hunk 2]
```diff
@@ -128,7 +128,7 @@ Not all installation options are supported for all platforms, as shown in the fo
 //This table is for all flavors of OpenShift, except OKD. A separate table is required because OKD does not support multiple AWS architecture types. Trying to maintain one table using conditions, while convenient, is very fragile and prone to publishing errors.
 ifndef::openshift-origin[]
 |===
-||Alibaba |AWS (x86_64) |AWS (arm64) |Azure (x86_64) |Azure (arm64)|Azure Stack Hub |GCP |Nutanix |{rh-openstack} |RHV |Bare metal (x86_64) |Bare metal (arm64) |vSphere |VMC |IBM Cloud VPC |{ibmzProductName} |{ibmpowerProductName}
+||Alibaba |AWS(64-bit x86) |AWS(64-bit ARM) |Azure(64-bit x86) |Azure(64-bit ARM)|Azure Stack Hub |GCP |Nutanix |{rh-openstack} |RHV |Bare metal(64-bit x86) |Bare metal(64-bit ARM) |vSphere |VMC |IBM Cloud VPC |{ibmzProductName} |{ibmpowerProductName}
```
Comment (maxwelldb) [inline installing/installing-preparing.adoc:131]: Did you intend to omit a space between the platform name and the opening parens?

File: installing/installing-preparing.adoc [hunk 3]
```diff
@@ -459,7 +459,7 @@ endif::openshift-origin[]
 //This table is for all flavors of OpenShift, except OKD. A separate table is required because OKD does not support multiple AWS architecture types. Trying to maintain one table using conditions, while convenient, is very fragile and prone to publishing errors.
 ifndef::openshift-origin[]
 |===
-||Alibaba |AWS (x86_64) |AWS (arm64) |Azure |Azure Stack Hub |GCP |Nutanix |{rh-openstack} |RHV |Bare metal (x86_64) |Bare metal (arm64) |vSphere |VMC |IBM Cloud VPC |{ibmzProductName} |{ibmzProductName} with {op-system-base} KVM |{ibmpowerProductName} |Platform agnostic
+||Alibaba |AWS(64-bit x86) |AWS(64-bit ARM) |Azure |Azure Stack Hub |GCP |Nutanix |{rh-openstack} |RHV |Bare metal(64-bit x86) |Bare metal(64-bit ARM) |vSphere |VMC |IBM Cloud VPC |{ibmzProductName} |{ibmzProductName} with {op-system-base} KVM |{ibmpowerProductName} |Platform agnostic
```
Comment (maxwelldb) [inline installing/installing-preparing.adoc:462]: Another:) 

Did you intend to omit a space between the platform name and the opening parens?

**Review by maxwelldb — COMMENTED** [review]
One question (x2).

Also, to double-check: this is 4.12 only, right? Wanted to make sure due to the 4.10+ and 4.11+ preview labels.

FYI @ShaunaDiaz

File: installing/installing-preparing.adoc (hunk 2 above)
Comment (kelbrown20) [reply installing/installing-preparing.adoc:131]: Ooo, great catch! There is suppose to be a space between the platform name and the opening parenth. Will fix that! Thank you!

**Pull request merged**